e2e/setup_cluster.sh

```diff
@@ -39,6 +39,14 @@ containerdConfigPatches:
 nodes:
   - role: control-plane
   - role: worker
+    kubeadmConfigPatches:
+    - |
+      kind: InitConfiguration
+      nodeRegistration:
+        kubeletExtraArgs:
+          pod-manifest-path: "/etc/kubernetes/manifests/"
+          cluster-dns: "10.254.0.10"
+          cluster-domain: "kube.local"
```

Comment:
'cluster-dns' and 'cluster-domain' might not required I suppose (pod-manifest-path is only required).
Is there any reason to add these?

Reply:
Those params are in the static pod quickstart guide from official k8s docs. I'll try running the test without these parameters and see what happens...

Reply:
Seems like it works fine without, as you suggested.